pkg/hns/netconf_windows.go

```diff
@@ -54,42 +55,74 @@ type PortMapEntry struct {
 	HostIP        string `json:"hostIP,omitempty"`
 }
 
+func (p *PortMapEntry) GetProtocolEnum() (uint32, error) {
```

Comment:
Type defining the Protocols as uint32 and giving a list of constants might be preferred

Reply:
updated